packages/vscode-extension/src/treeview/environmentTreeItem.ts

```diff
@@ -144,11 +158,18 @@ export class EnvironmentNode extends DynamicNode {
 }
 
 class WarningNode extends DynamicNode {
-  constructor(public identifier: string, warnings: string[]) {
+  constructor(public identifier: string, accountStatus: accountStatus) {
     super(identifier, vscode.TreeItemCollapsibleState.None);
-    this.label = `Sign in with your correct Azure / M365 account`;
+    if (accountStatus.isAzureAccountLogin === false && !accountStatus.isM365AccountLogin) {
+      this.label = `Sign in with your correct Azure / M365 account`;
+    } else if (!accountStatus.isM365AccountLogin) {
+      this.label = `Sign in with your correct M365 account`;
+    } else if (!accountStatus.isAzureAccountLogin === false) {
```

Comment:
Should be "accountStatus.isAzureAccountLogin === false" here?

Reply:
nice catch!!